src/components/Explore/MultipleLocationsChart.tsx

```diff
@@ -263,15 +265,19 @@ function formatCurrentValueLabels(
   seriesList: Series[],
   getXPosition: (p: Column) => number,
   getYPosition: (p: Column) => number,
+  height: number,
 ): LabelInfo[] {
   return seriesList.reduce((labelInfoList: LabelInfo[], currSeries: Series) => {
     const lastPoint = last(currSeries.data);
-    const currMaxY = max(labelInfoList.map(info => info.y)) || 0;
+    const currMinY = min(labelInfoList.map(info => info.y)) || height;
     return [
       ...labelInfoList,
       {
-        x: lastPoint?.x ? getXPosition(lastPoint) : 0,
-        y: lastPoint?.y ? Math.max(getYPosition(lastPoint), currMaxY + 14) : 0,
+        x: lastPoint?.x != null ? getXPosition(lastPoint) : 0,
+        y:
+          lastPoint?.y != null
+            ? Math.min(getYPosition(lastPoint), currMinY - 14)
+            : 0,
```

Comment:
Had to add `!= null` because having a y value of `0` is expected and we want to position it normally.